### PR TITLE
task 10

### PR DESCRIPTION
**Pull request by PriyankaChaganti**

### CODE REVIEW

File: robust_svm/read_images.py [hunk 1]
```diff
@@ -1,12 +1,43 @@
...
+                    annData.append(row)
+                count = count + 1
+        print(annData)
+
+
+
+
 
 
     """
```
Comment (sahithi403) [inline robust_svm/read_images.py]: This is not where the docstrings should be

File: robust_svm/read_images.py [hunk 2]
```diff
@@ -1,12 +1,43 @@
...
 
-def make_dataset(dat_set_path,data_set_list)
+def make_dataset(dat_set_path,data_set_list):
     """
     :param dat_set_path: path to datasets(training/test)
     :param data_set_list: folders list
     :return: location
     """
 def read_image_annotations(data_path,data_set_name):
+    dataSetList = ['00013', '00014', '00015', '00017', '00019']
```
Comment (sahithi403) [inline robust_svm/read_images.py]: Remove lines 17-22. You can't hardcode paths. @PriyankaChaganti

File: robust_svm/read_images.py (hunk 1 above)
Comment (PriyankaChaganti) [reply robust_svm/read_images.py]: do u want me to add docstrings at the beginning of the function

File: robust_svm/read_images.py [hunk 3]
```diff
@@ -15,7 +46,10 @@ def read_image_annotations(data_path,data_set_name):
     :param data_set_name: 
     :return: 
     """
-def get_cropped_image(file_path,annotation)
+def get_cropped_image(file_path,annotation):
+
+
+
     """
```
Comment (sahithi403) [inline robust_svm/read_images.py:43]: At the end of the file add the following snipppet

`if __name__ == "__main__":
datasets_path='data/training_data/Images/'
dataset_name = '00013'
read_image_annotations(datasets_path, dataset_name)`

File: robust_svm/read_images.py (hunk 1 above)
Comment (sahithi403) [reply robust_svm/read_images.py]: yes

File: robust_svm/read_images.py (hunk 3 above)
Comment (PriyankaChaganti) [reply robust_svm/read_images.py:43]: in main??

Comment (sahithi403) [reply robust_svm/read_images.py:43]: no. robust_svm/read_images.py

File: robust_svm/read_images.py [hunk 4]
```diff
@@ -1,24 +1,60 @@
...
+    :param data_set_name:
+    :return:
     """
-def get_cropped_image(file_path,annotation)
+    dataSetList = ['00013', '00014', '00015', '00017', '00019']
+
+    dataset = '00000'
+    dataloc = 'C:/Users/ch.srivamsi priyanka/Documents/GitHub/traffic_sign_svm/data/training_data/Images/'
+
+    for dataset in dataSetList:
```
Comment (sahithi403) [inline robust_svm/read_images.py]: Change the function logic so that it is only dependant on two parameters : datasets_path and dataset_name

**Comment by sahithi403** [comment]
Your branch has conflicts. Pull changes from development into your branch

**Pull request merged**